### ISSUE TITLE
Plugin listing ignores bottom padding in FireFox

### ISSUE THREAD

**Issue by JayBox325** [craftcms/cms#3874]
### Description
Nothing major at all. But when viewing the list of plugins in my project in FireFox the bottom padding is overlapped by the table within.

Could just add a bottom margin to the table#plugins element?

![screenshot 2019-02-20 at 20 19 35](https://user-images.githubusercontent.com/13233809/53122101-ac3ddc80-354d-11e9-820b-ad0c33873d78.png)

I notice it doesn't overlap on Mobile/Tablet sizes, only desktop.

### Additional info

- Craft version: 3.1.12
- Firefox version: 65.0.1

**Comment by brandonkelly** [comment]
Well that’s awkward. Seems like a Firefox bug though – nothing is telling it to ignore the bottom padding on `#content`.

**Comment by magnessjo** [comment]
@brandonkelly, 

  I took a stab at fixing this while I had some down time : [PR](https://github.com/craftcms/cms/pull/3905).  

  It’s an issue on every page that has the #content element (not just the plugin page).  You can reproduce it by shrinking your window to ~500px.  

  It might not the ideal solution but it will hopefully point you guys in the right direction.

**Comment by brandonkelly** [comment]
This has been fixed for Craft 3.4. (See https://github.com/craftcms/cms/issues/2883#issuecomment-554751414 for more details.)

**Comment by JayBox325** [comment]
I've just installed it and this is much better. Thanks. I just upped it to 48px and it looks a bit neater giving the final panel some more breathing room away from the bottom of the screen.

The page ended with a matrix where I viewed this and it did seem a bit tucked away and benefited from a little bit extra to bring it up into the page.

But this is obviously just my opinion and not a change request. Happy with it how it is! Great job.
